tools/codegen/gen.py

```diff
@@ -553,11 +553,11 @@ def compute_cpp_argument_yaml(cpp_a: Binding, *, schema_order: bool, kwarg_only_
         raise AssertionError()
     elif isinstance(cpp_a.argument, Argument):
         return compute_argument_yaml(
-            cpp_a.argument, schema_order=schema_order,
-            kwarg_only_set=kwarg_only_set, out_arg_set=out_arg_set, name_to_field_name=name_to_field_name)
+            cpp_a.argument, schema_order=schema_order, kwarg_only_set=kwarg_only_set,
+            out_arg_set=out_arg_set, name_to_field_name=name_to_field_name)
 
-def compute_argument_yaml(a: Argument, *, schema_order: bool, kwarg_only_set: Set[str],
-                          out_arg_set: Set[str], name_to_field_name: Dict[str, str]) -> object:
+def compute_argument_yaml(a: Argument, *, schema_order: bool, kwarg_only_set: Set[str], out_arg_set: Set[str],
+                          name_to_field_name: Dict[str, str]) -> object:
```

Comment:
Did anything happen here?

Reply:
Nope. I think I previously added an argument to these functions and later removed it, but forgot to revert the formatting changes that I made.

It's confusing in this PR, so I'm just going to fix it